lib/std/meta.zig

```diff
@@ -103,6 +103,70 @@ test "std.meta.stringToEnum" {
     testing.expect(null == stringToEnum(E1, "C"));
 }
 
+pub fn errorInSet(err: anytype, comptime Set: type) bool {
```

Comment:
I added this utility function. I think it is generally useful. I could also move this into a separate pr if that would be easier to review.